### PR TITLE
Started ex 29 but tischtennis is calling now 

### PR DESCRIPTION
**Pull request by AndreaBLS**

### CODE REVIEW

File: 29-PB-functions/29-index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const calculateBaseToExponent = (baseNumber = 0, exponentNumber = 0) =>
+    baseNumber ** exponentNumber;
+
+console.log("Task 6:")
+console.log(calculateBaseToExponent(5, 5));
+console.log(calculateBaseToExponent(10, 10));
+console.log(calculateBaseToExponent(3, 3));
+
+// Task 7
+const humanYears = 7;
```
Comment (martl) [inline 29-PB-functions/29-index.js:69]: you can put that const inside the function

File: 29-PB-functions/29-index.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        totalAmountNeeded = ((remainingYears * 365) * amountPerDay);
+    return `You will need ${totalAmountNeeded} of whatever you want to eat (${amountPerDay} a day) to last till the age of ${maxage} `;
+}
+console.log("Task 8:");
+console.log(calcSupply(25, 2));
+console.log(calcSupply(40, 3));
+
+// Task 9 
+const isWaldoHere = (string) => {
+    return string.includes('Waldo')
```
Comment (martl) [inline 29-PB-functions/29-index.js:89]: semicolon

File: 29-PB-functions/29-index.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+const humanYears = 7;
+const dogAge = (puppyAge = 0) =>
+    `Your doggo is ${puppyAge*humanYears} years old in human years!`
+
+console.log("Task 7:");
+console.log(dogAge(4));
+
+// Task 8 
+const calcSupply = (actualAge = 0, amountPerDay = 0) => {
+    const maxage = 80,
```
Comment (martl) [inline 29-PB-functions/29-index.js:78]: easier to read maxAge

File: 29-PB-functions/29-index.js (hunk 1 above)
Comment (AndreaBLS) [reply 29-PB-functions/29-index.js:69]: it doesnt work anymore if i do so, because of the arrow function i guess?

**Pull request closed without merging**